### PR TITLE
lug: add lint for config

### PR DESCRIPTION
**Pull request by skyzh**

### CODE REVIEW

File: lug/config.bootstrap.yaml [hunk 1]
```diff
@@ -1,166 +1,160 @@
...
   address: :7001
-
-dummy:
-  oneshot_common: &oneshot_common
+loglevel: 4
+repos:
+  - name: centos
+    path: /srv/centos
     retry: 5
     retry_interval: 60
```
Comment (doctormin) [inline lug/config.bootstrap.yaml:15]: missing of `interval` (seems to be my fault)

**Review by doctormin — CHANGES_REQUESTED** [review]
Rest LGTM

File: lug/config.bootstrap.yaml (hunk 1 above)
Comment (skyzh) [reply lug/config.bootstrap.yaml:15]: Just found out that my pyaml format embeds the dummy section into the file. This is not expected. I'll work on this later.

**Review by doctormin — APPROVED** [review]
LGTM

**Pull request merged**